app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,13 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
Các methods mà ko có phần xử lý gì thì e viết như cú pháp bên dưới nhé
```suggestion
  def home;end
```